winit/src/application/state.rs

```diff
@@ -36,6 +36,8 @@ impl<A: Application> State<A> {
             )
         };
 
+        window.set_menu(Some(conversion::menu(application.menu())));
```

Comment:
At this point we want to store the `Menu` in the `State`, should the `State` hold a map for the `Message`s produced by the `Menu` or should the `Menu` do it, how do we actually produce those without using `Clone`?